aspnetcore/release-notes/aspnetcore-10/includes/MinApiEmptyStringInFormPost.md

```diff
@@ -0,0 +1,25 @@
+### Treating empty string in form post as null for nullable value types
+
+When using the `[FromForm]` attribute with a complex object in minimal APIs, empty string values in a form post are now converted to null rather than causing a parse failure. This behavior matches the processing logic for form posts not associated with complex object's in minimal APIs.
+
+```csharp
+using Microsoft.AspNetCore.Http;
+
+var builder = WebApplication.CreateBuilder(args);
+
+var app = builder.Build();
+
+app.MapPost("/todo", ([FromForm] Todo todo) => TypedResults.Ok(todo));
+
+app.Run();
+
+public class Todo
+{
+  public int Id { get; set; }
+  public DateOnly? DueDate { get; set; } // Empty strings map to `null`
+  public string Title { get; set; }
+  public bool IsCompleted { get; set; }
+}
+```
+
+Thanks to @nvmkpk for contributing this change!
```

Comment:
@wadepickett  where is the issue to migrate this to the API docs?

Reply:
@Rick-Anderson, sorry, I didn't see your note was added after the PR merged.

This was just a prep PR to work from so reviewers including the dev team could see exactly what was changed afterwards.   The changes to this prep occurred in PR #34990.  This was all for the preview release in March.

#34724 is for April which just started today, for which all snippets that end up in the What's New moves out to projects, not just this one.  So for example, I wouldn't expect you to note that in your What's new PR #34745 that had inline code left in, or for it to be noted every single other PR that left code inline.